modules/itc/src/main/scala/lucuma/itc/legacy/ItcRemoteGraphResult.scala

```diff
@@ -4,14 +4,19 @@
 package lucuma.itc.legacy
 
 import cats.data.NonEmptyList
+import cats.syntax.all.*
 import io.circe.Decoder
 import lucuma.itc.ItcChartGroup
 
-case class ItcRemoteGraphResult(
+case class ItcRemoteResult(
   versionToken: String,
   ccds:         NonEmptyList[ItcRemoteCcd],
   groups:       NonEmptyList[ItcChartGroup]
-)
+) {
+  def maxTotalSNRatio: Double = ccds.map(_.totalSNRatio).maximum
+  def maxWellDepth: Double    = ccds.map(_.wellDepth).maximum
+  def maxPeakPixelFlux: Int   = ccds.map(_.peakPixelFlux).maximum.toInt
```

Comment:
Would we benefit from (`lazy`?) `val`s here?

Reply:
It is probably not very critical, ccds are just a little list but wouldn't hurt
Most of the cost goes to processing the graph data